rg_shipit/rg_shipit.php

```diff
@@ -929,15 +931,17 @@ public function hookDisplayAdminOrder($params) {
         $customer->email,
         ($address->phone_mobile ? $address->phone_mobile : $address->phone)
       );
-      $insurance = new ShipitInsurance(0.0,'','',false);            
       $arrayProducts = array();
       $items = 0;
+      $insuranceProducts = '';
       foreach ($products as $prod) {
         $product = new ShipitProduct($prod['product_reference'],$prod['product_quantity'],null,null);
         array_push($arrayProducts,$product);
         $items++;
+        if ($insuranceProducts!='') $insuranceProducts .= ',';
```

Comment:
```suggestion
        if ($insuranceProducts != '') $insuranceProducts .= ',';
```

can be null insuranceProducts?